code/modules/admin/verbs/randomverbs.dm

```diff
@@ -549,6 +549,32 @@ Traitors and the like can also be revived with the previous role mostly intact.
 	admin_ticket_log(M, msg)
 	SSblackbox.record_feedback("tally", "admin_verb", 1, "Rejuvinate") //If you are copy-pasting this, ensure the 2nd parameter is unique to the new proc!
 
+/client/proc/cmd_admin_freeze(mob/living/M in GLOB.mob_list)
+	set category = "Special Verbs"
+	set name = "Freeze Player"
+	if(!holder)
+		to_chat(src, "Only administrators may use this command.")
+		return
+	if(!istype(M))
+		return
+
+	if(!M.adminfrozen)
+		var/sleepies = M.AmountSleeping()
+		M.adminfrozen = sleepies ? sleepies : 1
+		M.SetSleeping(200000 SECONDS)//20k seconds to get your admin shit together
+		M.adminfreezeoverlay = new()
+		M.add_overlay(M.adminfreezeoverlay)
+		M.anchored = TRUE
+		log_admin("[key_name(usr)] froze [key_name(M)]!")
+		to_chat(M, "<span class='userdanger'>You have been frozen by Administrator [usr.key]!</span>")
```

Comment:
This should probably respect envision the same way that normal tickets do.

Reply:
> This should probably respect envision the same way that normal tickets do.

I don't understand what that means.

Reply:
Invismin*

Reply:
> Invismin*

I checked trough the adminhelp and who code but couldn't find anything regarding invismin, can you lend a hand here?